meta-facebook/sb-rb/src/platform/plat_cpld.c

```diff
@@ -0,0 +1,15 @@
+#include "plat_cpld.h"
+#include "libutil.h"
+#include "plat_i2c.h"
+#include "plat_util.h"
+#include <logging/log.h>
+
+#define AEGIS_CPLD_ADDR (0x4C >> 1)
```

Comment:
RB?

Reply:
fix in https://github.com/facebook/OpenBIC/commit/1ca77bdef1cd084143641d151f9b2d9fc256f8ee